apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -606,6 +589,22 @@ function initializeBlocklyWrapper(blocklyInstance) {
     blocklyWrapper.isStartMode = !!opt_options.editBlocks;
     const workspace = blocklyWrapper.blockly_.inject(container, options);
 
+    // When a block is clicked, dragged or deleted, we remove any "Unused clock" frame.
```

Comment:
Would `cdoUtils.js` be an appropriate place to defined the function below?

Reply:
Good suggestion! I think we do need to do the actual attaching here because we need to add the listener right away once we have a workspace (so, after `inject()` is called). I moved the the listener function into `blockSvgUnused.js` which is where we define the svg that we're interested in disposing here. I also added a comment explaining its purpose.
https://github.com/code-dot-org/code-dot-org/pull/51611/commits/e8c049d020e6d86f11393eefb04c01f289be71e9